src/ImageSharp/PixelAccessor.cs

```diff
@@ -110,6 +110,26 @@ protected override void CopyToZYX(PixelArea<Color> area, int sourceY, int source
         }
 
         /// <inheritdoc />
+        protected override unsafe void CopyToXYZ(PixelArea<Color> area, int sourceY, int sourceX, int width, int height)
```

Comment:
Ha! I actually wrote this myself the other day then never pushed it. Great minds.....